### PR TITLE
spring-annotation plugin add component

### PR DESCRIPTION
**Pull request by LiWenGu**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [X] New feature provided
- [ ] Improve performance

- Related issues

#5157 
___
### New feature or improvement
- Describe the details and related test reports.
spring-annotation plugin add component

### CODE REVIEW

File: oap-server/server-core/src/test/resources/component-libraries.yml [hunk 1]
```diff
@@ -173,8 +176,26 @@ Elasticsearch:
 transport-client:
   id: 48
   languages: Java
-Undertow:
+http:
   id: 49
+  languages: Java,C#,Node.js
```
Comment (wu-sheng) [inline oap-server/server-core/src/test/resources/component-libraries.yml:181]: Why change this file?

Comment (LiWenGu) [reply oap-server/server-core/src/test/resources/component-libraries.yml:181]: thiis file is sync from oap-server/server-bootstrap/src/main/resources/component-libraries.yml. Maybe this pr should not make too many changes to the file?

Comment (LiWenGu) [reply oap-server/server-core/src/test/resources/component-libraries.yml:181]: > Why change this file?

Do I need to reset this file(just change what I need to change)?

Comment (wu-sheng) [reply oap-server/server-core/src/test/resources/component-libraries.yml:181]: This is just a UT related file. You just need to change the one in the src/main/resources folder.

**Pull request merged**